test-runner-frontend/src/Views/Report.js

```diff
@@ -0,0 +1,390 @@
+    dispatch({
+      type: 'handleSnackBar',
+      payload: { transition: Transition, message: message, isOpen: true, messageDuration: messageDuration },
+    });
+    downloadJSON(jsonResult.className, jsonResult.jobId, jsonResult.jsonResults);
+  };
+
+  const downloadPDF = (Transition, message, messageDuration) => {
+    dispatch({
+      type: 'handleSnackBar',
+      payload: { transition: Transition, message: message, isOpen: true, messageDuration: messageDuration },
+    });
+    printPDF();
+  };
+
+  return (
+    <div>
+      {loadingState === true ? (
+        <Box
+          sx={{
+            position: 'absolute',
+            top: '45%',
+            left: '50%',
+            marginTop: 0,
+            marginLeft: 0,
+          }}
+        >
+          <CircularProgress style={{ width: '10vh' }} color="secondary" />
+        </Box>
+      ) : (
+        <div>
+          <img
+            style={{
+              marginLeft: '2%',
+              marginTop: '0.5%',
+              marginBottom: '1%',
+              width: '18%',
+            }}
+            src={GraphQL_Logo}
+            alt="GraphQL Java Logo"
+          />
+          <a
+            style={{
+              marginRight: '3.5%',
+              marginTop: '1.1%',
+              float: 'right',
+              textDecoration: 'none',
+              color: 'gray',
+              fontWeight: '720',
+            }}
+            href={GRAPHQL_JAVA_GITHUB}
+          >
+            GitHub
+          </a>
+
+          <Dialog
+            classes={{
+              paper: classes.dialog,
+            }}
+            open={openDialog}
+            onClose={() => handleCloseDialog()}
+          >
+            <AppBar sx={{ position: 'relative', bgcolor: 'transparent' }}>
+              <Toolbar>
+                <Typography sx={{ ml: 2, flex: 1, color: 'black' }} variant="h6" component="div">
+                  <b>{jsonResult.className}</b>
+                </Typography>
+                <IconButton color="inherit" onClick={() => handleCloseDialog()} sx={{ color: 'gray' }}>
+                  <CloseIcon />
+                </IconButton>
+              </Toolbar>
+
+              <Stack direction="row" spacing={2} style={{ marginLeft: '6%', marginBottom: '3%' }}>
+                <Button
+                  variant="outlined"
+                  sx={{ color: '#e535ab', borderColor: '#e535ab' }}
+                  onClick={() => downloadJSONReport(TransitionSnackBar, 'JSON file downloaded', null)}
+                >
+                  Download
+                </Button>
+                <Button
+                  variant="outlined"
+                  sx={{ color: 'gray', borderColor: 'gray' }}
+                  onClick={handleClickOpenSnackBar(TransitionSnackBar, 'JSON results saved into clipboard', null)}
+                >
+                  Copy
+                </Button>
+              </Stack>
+            </AppBar>
+
+            <List>
+              <ListItem>
```

Comment:
im not sure a list is the correct type of element to use here

Reply:
Done, I changed it to a Box.